api/routes/ws.dart

```diff
@@ -48,7 +48,8 @@ Future<Response> onRequest(RequestContext context) async {
               sessionBloc.add(OnMessageSent(chatModel));
 
             case AddPlayerEvent:
-              player.name = (websocketEvent as AddPlayerEvent).data;
+              player.name = (websocketEvent as AddPlayerEvent).data['name']!;
```

Comment:
if `AddPlayerEvent` extends `WebSocketEvent<Player>`
then this would be simple:
`player=(websocketEvent as AddPlayerEvent).data;`